modules/ROOT/pages/transaction-management.adoc

```diff
@@ -146,6 +145,103 @@ When running in a transactional scope, the entire process must run in the same t
 
 Any other router or scope (for example, Foreach) continues with the transaction and its behavior is not modified.
 
+== Error Handling
+
+When working with transactions, we need to know how to handle errors. This includes to continue with the transaction even if an specific error occurs, or to handle correctly the rollback. 
+
+As mentioned in the xref:on-error-scope-concept.adoc[On-Error Scope], there are two types of Error Handlers, which relate differently with Transactions:
+
+* On Error Propagate: If the `on-error-propagate` is in the Error Handler corresponding to the component that began the Transaction, then it rollbacks it **before executing the inner processors of the On Error Propagate Scope**. Thus, the inner processor do not run within the transaction. If the `on-error-propagate` is inside an inner element that did not start the Transaction, then it is not rolled back and the inner processors of such `on-error-propagate` remain running within the Transaction. Remember that running within a Transaction affect different scopes and routers. 
```

Comment:
general comment about the paragraph: it is not clear what `inner` is relative to. Can you rephrase?

Reply:
```suggestion
* On Error Propagate: If the `on-error-propagate` is in the Error Handler corresponding to the component that began the Transaction, then it rolls it backs **before executing the processors of the On Error Propagate Scope**. Thus, those processors inside the error handler do not run within the transaction. If the `on-error-propagate` is inside an inner element that did not start the Transaction, then it is not rolled back and the inner processors of such `on-error-propagate` remain running within the Transaction. Remember that running within a Transaction affect different scopes and routers. 
```